src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/MSSqlServerSink.cs

```diff
@@ -286,9 +262,7 @@ void FillDataTable(IEnumerable<LogEvent> events)
                             row["Level"] = logEvent.Level;
                             break;
                         case StandardColumn.TimeStamp:
-                            row["TimeStamp"] = _columnOptions.TimeStamp.ConvertToUtc
-                                ? logEvent.Timestamp.DateTime.ToUniversalTime()
-                                : logEvent.Timestamp.DateTime;
+                            row["TimeStamp"] = _columnOptions.TimeStamp.ConvertToUtc ? logEvent.Timestamp.DateTime.ToUniversalTime() : logEvent.Timestamp.DateTime;
```

Comment:
and the rest of the diffs look like some whitespace and formatting changes slipped in.